tools/deploy-with-olm.sh

```diff
@@ -41,19 +41,20 @@ TARGET_NAMESPACE=${TARGET_NAMESPACE:-openshift-storage}
 
 # CSV environment variables
 ADDON_NAME=${ADDON_NAME:-ocs-converged}
-BUNDLE_FILE=bundle/manifests/ocs-osd-deployer.clusterserviceversion.yaml
+BUNDLE_FILE=manifests/ocs-osd-deployer.clusterserviceversion.yaml
 CLUSTER_SIZE=${CLUSTER_SIZE:-1}
 
 # Generate the deployer image 
 blue "Generate deployer image: ${DEPLOYER_IMAGE}"
 exit_on_err make docker-build IMG=${DEPLOYER_IMAGE}
 exit_on_err make docker-push IMG=${DEPLOYER_IMAGE}
 
+OUTPUT_DIR=./tools/bundle
 # Generate the olm bundle image
 blue "Generate and push the olm bundle image: ${BUNDLE_IMAGE}"
-exit_on_err make bundle IMG=${DEPLOYER_IMAGE} OUTPUT_DIR=./tools/bundle
-exit_on_err sed -i "s|- name: ADDON_NAME|- {name: ADDON_NAME, value: ${ADDON_NAME}}|" ${BUNDLE_FILE}
-exit_on_err sed -i "s|- name: SOP_ENDPOINT|- {name: SOP_ENDPOINT, value: \'${SOP_ENDPOINT}\'}|" ${BUNDLE_FILE}
+exit_on_err make bundle IMG=${DEPLOYER_IMAGE} OUTPUT_DIR=${OUTPUT_DIR}
+exit_on_err sed -i "s|- name: ADDON_NAME|- {name: ADDON_NAME, value: ${ADDON_NAME}}|" ${OUTPUT_DIR}/${BUNDLE_FILE}
+exit_on_err sed -i "s|- name: SOP_ENDPOINT|- {name: SOP_ENDPOINT, value: \'${SOP_ENDPOINT}\'}|" ${OUTPUT_DIR}/${BUNDLE_FILE}
```

Comment:
Why do we need to `BUNDLE_FILE` separately then `OUTPUT_DIR`? we never use it on its own:

```suggestion
OUTPUT_DIR=./tools/bundle
BUNDLE_FILE=${OUTPUT_DIR}/manifests/ocs-osd-deployer.clusterserviceversion.yaml
CLUSTER_SIZE=${CLUSTER_SIZE:-1}

# Generate the deployer image 
blue "Generate deployer image: ${DEPLOYER_IMAGE}"
exit_on_err make docker-build IMG=${DEPLOYER_IMAGE}
exit_on_err make docker-push IMG=${DEPLOYER_IMAGE}

# Generate the olm bundle image
blue "Generate and push the olm bundle image: ${BUNDLE_IMAGE}"
exit_on_err make bundle IMG=${DEPLOYER_IMAGE} OUTPUT_DIR=${OUTPUT_DIR}
exit_on_err sed -i "s|- name: ADDON_NAME|- {name: ADDON_NAME, value: ${ADDON_NAME}}|" ${BUNDLE_FILE}
exit_on_err sed -i "s|- name: SOP_ENDPOINT|- {name: SOP_ENDPOINT, value: \'${SOP_ENDPOINT}\'}|" ${BUNDLE_FILE}
```

Reply:
Yes, thanks for pointing it out.Looks cleaner.Fixed in next update